From 4185406af5ce31c62d9fac0a352e3bc82e324b91 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@cloudera.com>
Date: Wed, 30 Jan 2013 23:55:46 -0800
Subject: [PATCH 1278/1357] MR1: Revert "CLOUDERA-BUILD. Fix DataChecksum API usage to reflect HADOOP-8700 changes"

This reverts commit be0ed1e32fe18f2b1ab327414fbbc8789bb1e0ce.
---
 .../org/apache/hadoop/mapred/IFileInputStream.java |    3 +--
 .../apache/hadoop/mapred/IFileOutputStream.java    |    3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileInputStream.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileInputStream.java
index 2febf25..5495e4a 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileInputStream.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileInputStream.java
@@ -83,8 +83,7 @@ class IFileInputStream extends InputStream {
   public IFileInputStream(InputStream in, long len, Configuration conf) {
     this.in = in;
     this.inFd = getFileDescriptorIfAvail(in);
-    sum = DataChecksum.newDataChecksum(
-        DataChecksum.Type.valueOf(DataChecksum.CHECKSUM_CRC32),
+    sum = DataChecksum.newDataChecksum(DataChecksum.CHECKSUM_CRC32, 
         Integer.MAX_VALUE);
     checksumSize = sum.getChecksumSize();
     length = len;
diff --git a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileOutputStream.java b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileOutputStream.java
index b1c9eb9..84e022a 100644
--- a/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileOutputStream.java
+++ b/hadoop-mapreduce1-project/src/mapred/org/apache/hadoop/mapred/IFileOutputStream.java
@@ -45,8 +45,7 @@ class IFileOutputStream extends FilterOutputStream {
    */
   public IFileOutputStream(OutputStream out) {
     super(out);
-    sum = DataChecksum.newDataChecksum(
-        DataChecksum.Type.valueOf(DataChecksum.CHECKSUM_CRC32),
+    sum = DataChecksum.newDataChecksum(DataChecksum.CHECKSUM_CRC32,
         Integer.MAX_VALUE);
     barray = new byte[sum.getChecksumSize()];
   }
-- 
1.7.0.4

